blog/docs/articles/demystifying-activator-on-path.md

```diff
@@ -0,0 +1,182 @@
+# Demystifying Activator on path
```

Comment:
after reading the full text, it might be good to add the generic concept before going into the details. Something like

The activator acts as a component on the data-path to enable traffic buffering when a service is scaled-to-zero. One lesser known feature of activator is, that it can act as a request buffer that handles back-pressure with the goal to not overload a Knative service. For this, a Knative service can define how much traffic it can handle using annotations (link to docs here). The autoscaler component will use this information to calculate the amount of pods needed to handle the incoming traffic for a specific Knative service.